caliban_toolbox/figure_eight_functions.py

```diff
@@ -129,6 +130,25 @@ def copy_job(job_id, key):
     return new_job_id
 
 
+def rename_job(job_id, key, name):
+    """Helper function to create a Figure 8 job based on existing job.
+        Args:
+            job_id: ID number of job to rename
+            key: API key to access Figure 8 account
+            name: new name for job
+        """
+
+    headers = {'content-type': 'application/json'}
+    payload = {
+        'key': key,
+        'job': {
+            'title': name
+        }}
+    response = requests.put(
+        'https://api.appen.com/v1/jobs/{}.json'.format(job_id), data=json.dumps(payload),
+        headers=headers)
```

Comment:
use `requests.put(url, json=payload)`

It will automatically format the JSON data and create the headers for you.